script/bootstrap.ps1

```diff
@@ -17,25 +19,36 @@
 # You should have received a copy of the GNU General Public License
 # along with Vega Strike.  If not, see <https://www.gnu.org/licenses/>.
 
+
 # You can customize this directory location if desired, but it should be
 # something very short. Otherwise, you will run into problems.
-Set-Variable -Name VCKPG_PARENT_DIR -Value "C:\Projects"
-Set-Variable -Name CMAKE_VERSION -Value "3.21.1"
+param(
+    [String]$VCPKG_PARENT_DIR = "C:\Projects"
+)
+
+Set-Variable -Name CMAKE_VERSION -Value "3.25.1"
 
 New-Item "$VCKPG_PARENT_DIR" -ItemType Directory -Force
 Push-Location "$VCKPG_PARENT_DIR"
-git clone https://github.com/Microsoft/vcpkg.git
-.\vcpkg\bootstrap-vcpkg.bat -disableMetrics
+git clone https://github.com/stephengtuggy/vcpkg-local.git ./v
+.\v\bootstrap-vcpkg.bat -disableMetrics
```

Comment:
I assume this is temporary while the service is down, no?

Reply:
> I assume this is temporary while the service is down, no?

@BenjamenMeyer Actually, no. This is intended to be a long-term solution. For the last several months -- well before the nasm.us issues -- I have found it necessary to use a fork-of-a-fork of vcpkg in order to get Vega Strike to build successfully on Windows. I do intend to keep this fork updated semi-regularly.

Reply:
@BenjamenMeyer working on that right now: https://github.com/vegastrike/vcpkg-local

Reply:
Continuing to have problems with the Windows builds. Now it's Python that isn't compiling, via vcpkg. (Annoying that we even have to compile it at all. Maybe there's another way; not sure.)